### PR TITLE
Adds West coast mesh from SF to LA

### PR DESCRIPTION
**Pull request by pwolfram**
Adds an example West coast and HI mesh

### CODE REVIEW

**Comment by pwolfram** [comment]
image

Note background land is just a quick ParaView plotting hack to show relative location to CA coast.

**Comment by pwolfram** [comment]
cc @mark-petersen, @sbrus89 this is just a mesh.  

Big thing to check is path naming: `compass/ocean/CArefined/120km3km/`.

Visual approval should be ok and we should all agree on `CArefined` tag at `ocean` top level.

**Comment by mark-petersen** [comment]
Could we change the path
`testing_and_setup/compass/ocean/CArefined/120km3km/build_mesh`
to
`testing_and_setup/compass/ocean/global_ocean/CA120to3/init`
Everything on a full sphere is in the `global_ocean` subdirectory.

**Comment by pwolfram** [comment]
cc @cajeffery

**Comment by pwolfram** [comment]
@mark-petersen, we aren't running a simulation so would 

`testing_and_setup/compass/ocean/global_ocean/CA120to3/build_mesh` work?

Otherwise, we would have an incomplete `init` and we are moving toward a separated mesh building step for other parts of the coastal code, to allow for faster iteration if the init doesn't explicitly depend on the mesh, which it usually doesn't.

**Comment by pwolfram** [comment]
cc @markgalassi

**Comment by sbrus89** [comment]
I like the `testing_and_setup/compass/ocean/global_ocean/CA120to3/build_mesh` path

File: testing_and_setup/compass/ocean/jigsaw_to_MPAS/coastal_tools.py [hunk 1]
```diff
@@ -72,6 +72,9 @@
                                        [-113.94, 30.77],
                                        [-119.44, 33.09]])]}
 
+SFtoLA        = {"include": [np.array([-124.0, -117.5, 34.2, 38.0])],  # SF to LA
+                 "exclude": [np.array([-122.1, -120.8, 37.7, 39.2])]}  # SF Bay Delta
+
```
[the remark this answers is not in the document]
Comment (pwolfram) [reply testing_and_setup/compass/ocean/jigsaw_to_MPAS/coastal_tools.py]: True @sbrus89.  Thoughts on whether we should have a global library in coastal_tools or in individual scripts @mark-petersen?  I'm inclined to move it to the script myself in accord with @sbrus89.

[the remark this answers is not in the document]
Comment (sbrus89) [reply testing_and_setup/compass/ocean/jigsaw_to_MPAS/coastal_tools.py]: For now I think that's the way to go. Long term, we should probably create a library similar to `geometric_features`

[the remark this answers is not in the document]
Comment (pwolfram) [reply testing_and_setup/compass/ocean/jigsaw_to_MPAS/coastal_tools.py]: No modification of this file now... I agree that this information should probably go into geometric features and be directly used from there so GIS can be leveraged but that is beyond the scope of this PR for now.

**Comment by pwolfram** [comment]
image
HI patch-based mesh example.

**Comment by pwolfram** [comment]
@mark-petersen, @sbrus89, @cajeffrey, do you have additional recommendations before a merge?

**Comment by pwolfram** [comment]
Thanks, we can update a minor fix in a future PR as needed.  This way folks can use the `ocean/coastal` branch to move forward.

**Pull request merged**